### ISSUE TITLE
pip installed nbformat 5.6.0 reports 0.0.0 version number on Linux

### ISSUE THREAD

**Issue by yxlao** [jupyter/nbformat#306]
### `nbformat==5.6.0` reports version number as `0.0.0`
```bash
(testpy37) ➜  ~ pip install nbformat==5.6.0 --no-cache-dir              
Collecting nbformat==5.6.0
  Downloading nbformat-5.6.0-py3-none-any.whl (77 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 77.0/77.0 kB 5.4 MB/s eta 0:00:00
Collecting jupyter-core
  Downloading jupyter_core-4.11.1-py3-none-any.whl (88 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 88.4/88.4 kB 27.3 MB/s eta 0:00:00
Collecting fastjsonschema
  Downloading fastjsonschema-2.16.2-py3-none-any.whl (22 kB)
Collecting traitlets>=5.1
  Downloading traitlets-5.4.0-py3-none-any.whl (107 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.1/107.1 kB 52.3 MB/s eta 0:00:00
Collecting jsonschema>=2.6
  Downloading jsonschema-4.16.0-py3-none-any.whl (83 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 83.1/83.1 kB 45.0 MB/s eta 0:00:00
Collecting importlib-resources>=1.4.0
  Downloading importlib_resources-5.9.0-py3-none-any.whl (33 kB)
Collecting importlib-metadata
  Downloading importlib_metadata-4.12.0-py3-none-any.whl (21 kB)
Collecting pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0
  Downloading pyrsistent-0.18.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (117 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 117.1/117.1 kB 105.4 MB/s eta 0:00:00
Collecting pkgutil-resolve-name>=1.3.10
  Downloading pkgutil_resolve_name-1.3.10-py3-none-any.whl (4.7 kB)
Collecting typing-extensions
  Downloading typing_extensions-4.3.0-py3-none-any.whl (25 kB)
Collecting attrs>=17.4.0
  Downloading attrs-22.1.0-py2.py3-none-any.whl (58 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.8/58.8 kB 450.9 MB/s eta 0:00:00
Collecting zipp>=3.1.0
  Downloading zipp-3.8.1-py3-none-any.whl (5.6 kB)
Installing collected packages: fastjsonschema, zipp, typing-extensions, traitlets, pyrsistent, pkgutil-resolve-name, attrs, jupyter-core, importlib-resources, importlib-metadata, jsonschema, nbformat
Successfully installed attrs-22.1.0 fastjsonschema-2.16.2 importlib-metadata-4.12.0 importlib-resources-5.9.0 jsonschema-4.16.0 jupyter-core-4.11.1 nbformat-5.6.0 pkgutil-resolve-name-1.3.10 pyrsistent-0.18.1 traitlets-5.4.0 typing-extensions-4.3.0 zipp-3.8.1

(testpy37) ➜  ~ python -c "import nbformat; print(nbformat.__version__)"
0.0.0
```

### `nbformat==5.5.0` works
```bash
(testpy37) ➜  ~ pip install nbformat==5.5.0 --no-cache-dir             
Collecting nbformat==5.5.0
  Downloading nbformat-5.5.0-py3-none-any.whl (75 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 75.3/75.3 kB 3.6 MB/s eta 0:00:00
Requirement already satisfied: jsonschema>=2.6 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from nbformat==5.5.0) (4.16.0)
Requirement already satisfied: fastjsonschema in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from nbformat==5.5.0) (2.16.2)
Requirement already satisfied: jupyter_core in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from nbformat==5.5.0) (4.11.1)
Requirement already satisfied: traitlets>=5.1 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from nbformat==5.5.0) (5.4.0)
Requirement already satisfied: typing-extensions in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (4.3.0)
Requirement already satisfied: importlib-metadata in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (4.12.0)
Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (1.3.10)
Requirement already satisfied: attrs>=17.4.0 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (22.1.0)
Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (0.18.1)
Requirement already satisfied: importlib-resources>=1.4.0 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from jsonschema>=2.6->nbformat==5.5.0) (5.9.0)
Requirement already satisfied: zipp>=3.1.0 in ./miniconda3/envs/testpy37/lib/python3.7/site-packages (from importlib-resources>=1.4.0->jsonschema>=2.6->nbformat==5.5.0) (3.8.1)
Installing collected packages: nbformat
  Attempting uninstall: nbformat
    Found existing installation: nbformat 5.6.0
    Uninstalling nbformat-5.6.0:
      Successfully uninstalled nbformat-5.6.0
Successfully installed nbformat-5.5.0

(testpy37) ➜  ~ python -c "import nbformat; print(nbformat.__version__)"
5.5.0
```
### System environments

```bash
(testpy37) ➜  ~ uname -a
Linux 5.17.15-051715-generic #202206141358 SMP PREEMPT Tue Jun 14 18:53:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

(testpy37) ➜  ~ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy

(testpy37) ➜  ~ python --version                                                                                       
Python 3.7.13
```

This issue affects downstream applications, such as `plotly`, which depends on `nbformat`, e.g., in [this issue](https://github.com/isl-org/Open3D/pull/5544).

**Comment by jconwell** [comment]
Ran into the same issue. This breaks Plotly usage in notebooks as plotly checks nbformat version > 4.2.0. 
For now if you install nbformat==5.5.0 Plotly will work correctly.

**Comment by blink1073** [comment]
Fixed in 5.6.1
